version.txt

```diff
@@ -1 +1 @@
-1.0.3.4-SNAPSHOT
+1.0.5-SNAPSHOT
```

Comment:
This should be `1.0.5.0-SNAPSHOT`

The pattern we've been using is ${dropwizard-veresion}-${increment}